SEP.md

```diff
@@ -0,0 +1,74 @@
+# Stronghold Enhancement Proposal
+
+The scope of this document is to describe a process to enhance and extend the functionality of IOTA Stronghold software library. This process is strongly influenced by the rust RFC process. 
+
+
+## Process
+
+1. Look out for Issues
+    - The very fist step is to look out for issues, api enhancements, feature enhancements and feature proposals
+2. Is the Enhancement Propopsal really necessary?
+    - Does the Enhancement bring in any new features, or could this be a bugfix?
+3. Copy the template from below and fill in the necessary parts
+    - Put in a convincing motivation, a consise guide and implementation level explanation; if both parts are submitted poorly, this would clearly indicate a lack of understanding
```

Comment:
```suggestion
    - Put in a convincing motivation, a concise guide, and implementation level explanation; if both parts are submitted poorly, this would indicate a lack of understanding
```